index.html

```diff
@@ -0,0 +1,299 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio-Jorum Odiemo</title>
+    <link rel="stylesheet" href="styles.css">
+</head> 
+<body>
+    <header class="Top">
+        <div class="logo"><a href="index.html" class="logo">My Logo</a></div>
+        <nav class="menu-list">
+            <ul onclick = "hideMenu()">
+                <li><a href="#head" class="nav">Portfolio</a></li>
+                <li><a href="#about" class="nav">About</a></li>
+                <li><a href="#contact" class="nav">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="navigation"  onclick = "showMenu()">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+        </div>           
+    </header>
```

Comment:
- [ ] Kindly adjust the spacing and size of the mobile menu header on the mobile view to match the Figma design.

| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/175260659-b13bc06d-5122-459d-9b00-944be091268b.png) | ![image](https://user-images.githubusercontent.com/62290545/175260517-09dda81a-60a2-4860-a68e-bcb37b0a208b.png) |